package.json

```diff
@@ -28,7 +28,8 @@
     "start:server": "grunt clean:server && grunt start:server",
     "start:client": "grunt clean:client && grunt start:client",
     "build:server:offline": "cd ./api/cmd/portainer && CGO_ENABLED=0 go build -a --installsuffix cgo --ldflags '-s' && mv -f portainer ../../../dist/portainer",
-    "clean:all": "grunt clean:all"
+    "clean:all": "grunt clean:all",
+    "format": "prettier --loglevel warn --write \"**/*.{js,css,md}\""
```

Comment:
We'll want to work with the following pattern: `app/**/*.{js,css,html}`

(No need for markdown validation)